database/migrations/2020_03_22_1424_add_primary_key_to_device_graphs.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+    This migration adds primary key for device_graphs
+
+    Percona Xtradb refused to INSERT IGNORE into a table
+    without a primary key.
+ */
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class AddPrimaryKeyToDeviceGraphs extends Migration
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('device_graphs', function (Blueprint $table) {
+            $table->bigIncrements('id');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('device_graphs', function (Blueprint $table) {
+            $table->dropColumn('id');
```

Comment:
If we follow current IDs naming in LibreNMS, this one should be named `device_graphs_id`. What do you thing @murrant ?

Reply:
I've renamed the field as you've requested.

Reply:
@PipoCanaja that is the old naming scheme.  The old naming scheme prepended the table name to all fields to prevent collisions in joins.   When using Eloquent/Fluent it isn't needed.  I generally prefer `id` as that is the standard convention.